### PR TITLE
First Release of Ecowitt Weather Server

### PR DESCRIPTION
**Pull request by BJReplay**
This pull request completes most of the work to provide a local weather source based on #149 but doesn't close the issue:

- Dashboards still need to be modified to use the data
- The local weather server needs to be included into the docker stack; this can be done by including it in powerwall.yml - for example:

```
    localweather:
        build: ./localweather
        image: localweather:latest
        container_name: localweather
        hostname: localweather
        restart: always
        user: "1000:1000"
        volumes:
            - type: bind
              source: ./localweather
              target: /var/lib/localweather
              read_only: true
        ports:
            - target: 8686
              published: 8686
              mode: host
        environment:
            - WEATHERCONF=/var/lib/localweather/localweather.conf
        depends_on:
            - influxdb

```

### CODE REVIEW

File: weather/contrib/ecowitt/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# localweather Server
+
+![Docker Pulls](https://img.shields.io/docker/pulls/jasonacox/localweather)
+
+This server pulls current weather data from [Ecowitt.net Weather API](https://api.ecowitt.net/api/v3/device/real_time), makes it available via local API calls and stores the data in InfluxDB for graphing.
+
+This service was built to easily add weather data graphs to the [Powerwall-Dashboard](https://github.com/jasonacox/Powerwall-Dashboard) project.
+
+Docker: docker pull [jasonacox/localweather](https://hub.docker.com/r/jasonacox/localweather)
```
Comment (jasonacox) [inline weather/contrib/ecowitt/README.md:9]: Assuming others contribute similar services for other local weather stations (or services), I'm thinking we should probably rename the container to match the specific solution it supports.   What about just "ecowitt"?

**Comment by jasonacox** [comment]
Thanks @BJReplay  - excellent work!  Make sure your git commit email is set up as a github email so that you get credit as a contributor (if you care).  

This is a great addition! I added an in-line comment about the container name. I'm happy to publish that container to dockerhub, but want your thoughts on the name before I do.

File: weather/contrib/ecowitt/README.md (hunk 1 above)
Comment (BJReplay) [reply weather/contrib/ecowitt/README.md:9]: Agree that's a good idea.

Comment (BJReplay) [reply weather/contrib/ecowitt/README.md:9]: I'll make those changes tomorrow, and test, and assuming it all works, create another pull request.

**Pull request merged**